src/components/SoftwareWalletWarningMessage.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+
+/**
+ * Component that shows a warning message if user decides to use a software wallet
+ *
+ * @memberof Components
+ */
+const SoftwareWalletWarningMessage = (props) => {
+  return (
+    <div>
+      <p className="mt-4">Using a software wallet is not the safest method to store your tokens. Your data is exposed to an attack because it's connected to the internet.</p>
```

Comment:
Suggestion: "Using a software wallet is not the safest way to secure your tokens. As your seed words are stored in your computer, and your computer is connected to the internet, your seed words may be stolen by a virus."

Reply:
Done